app/router.go

```diff
@@ -2,12 +2,45 @@ package main
 
 import (
 	"net/http"
+	"net/url"
+	"strings"
 
 	"github.com/gorilla/mux"
 
 	"github.com/weaveworks/scope/render"
 )
 
+// customMatcher uses request.RequestURI (the raw, unparsed request) to attempt
+// to match pattern.  It does this as go's URL.Parse method is broken, and
+// mistakenly unescapes the Path before parsing it.  This breaks %2F (encoded
+// forward slashes) in the paths.
+func customMatcher(pattern string) mux.MatcherFunc {
```

Comment:
Is there any reason we *wouldn't* want to use this matcher for a route? Could it be HandleFunc(mux, path, handler)?